specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2018-09-01/containerregistry_build.json

```diff
@@ -1777,6 +1866,10 @@
           "description": "The Docker file path relative to the source location.",
           "type": "string"
         },
+        "target": {
+          "description": "Gets or sets the name of the target build stage for the docker build.",
```

Comment:
The "Gets or sets" is a C#-way of describing this field and is not applicable to all languages (i.e. not all languages generate getters and setters).  Consider rewording this description (there are two other cases where this phrasing is used).

Reply:
done